client/components/forms/form-textarea/index.jsx

```diff
@@ -2,7 +2,14 @@ import classnames from 'classnames';
 
 import './style.scss';
 
-const FormTextarea = ( { className, isError, isValid, children, forwardedRef, ...otherProps } ) => (
+const FormTextarea = ( {
+	className = '',
+	isError = false,
+	isValid = false,
+	children = undefined,
```

Comment:
Out of curiosity, what does setting the default to undefined accomplish?

Reply:
Setting a default value lets TS know that the prop is optional. Without these, this line shows a TS error:

https://github.com/Automattic/wp-calypso/blob/027010543fdbd8a99cfa4a55e54229ea8c6d732b/client/jetpack-cloud/sections/settings/advanced-credentials/credentials-form/index.tsx#L182

```
1. Type '{ name: string; id: string; value: string; onChange: FormEventHandler<HTMLInputElement>; disabled: boolean; className: string; isError: boolean | undefined; }' is missing the following properties from type '{ [x: string]: any; className?: string | undefined; isError?: boolean | undefined; isValid?: boolean | undefined; children: any; forwardedRef: any; }': children, forwardedRef
```